docs/api/commands/uncheck.mdx

```diff
@@ -57,11 +57,28 @@ Pass in an options object to change the default behavior of `.uncheck()`.
 | `timeout`                    | [`defaultCommandTimeout`](/guides/references/configuration#Timeouts)           | Time to wait for `.uncheck()` to resolve before [timing out](#Timeouts)                                                                            |
 | `waitForAnimations`          | [`waitForAnimations`](/guides/references/configuration#Actionability)          | Whether to wait for elements to [finish animating](/guides/core-concepts/interacting-with-elements#Animations) before executing the command.       |
 
+### Requirements [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Chains-of-Commands)
+
+- `.uncheck()` requires being chained off a command that yields DOM element(s).
+- `.uncheck()` requires the element to have type `checkbox` or `radio`.
+
+### Assertions [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Assertions)
+
+- Before executing, `.uncheck()` will wait for the element to reach an
```

Comment:
```suggestion
- Before executing, `.uncheck()` will wait for the subject to reach an
```

Reply:
Why would we use a more generic term here? We know it's an element(s), rather than any generic "subject" (this is also detailed above as one of the 'requirements' of the command).